.ci/pipeline/ci.yml

```diff
@@ -116,24 +116,21 @@ jobs:
   timeoutInMinutes: 120
   strategy:
     matrix:
-      Python3.9_Sklearn1.0:
```

Comment:
In my opinion, it would make sense to continue checking sklearn 1.0 plus the latest versions - any thoughts on dropping/keeping this?

Reply:
Updated.